app/src/main/java/com/example/androidacademyhomework/UI/MovieDetailsFragment.kt

```diff
@@ -21,12 +42,56 @@ class MovieDetailsFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        textViewDetailFragmentAgeLimit = view.findViewById(R.id.textViewDetailFragmentAgeLimit)
+        textViewDetailFragmentNameMove = view.findViewById(R.id.textViewDetailFragmentNameMove)
+        textViewDetailFragmentMovieGenre = view.findViewById(R.id.textViewDetailFragmentMovieGenre)
+        ratingBarDetailsFragment = view.findViewById(R.id.ratingBarDetailsFragment)
+        textViewDetailFragmentReviews = view.findViewById(R.id.textViewDetailFragmentReviews)
+        textViewDetailFragmentStory = view.findViewById(R.id.textViewDetailFragmentStory)
+        recyclerForActors = view.findViewById(R.id.recyclerForActors)
+        initRecycler()
         view.findViewById<Button>(R.id.buttonViewBack).setOnClickListener {
             returnToMovieListFragment()
         }
+        movieId = args.movieId
```

Comment:
I don't see a necessity to store id - you can always get it from args

Reply:
You're right. Fixed